.buildkite/longruns/pipeline.yml

```diff
@@ -124,10 +135,10 @@ steps:
           slurm_mem_per_cpu: 16G
           slurm_tasks_per_node: 2
 
-      - label: "Unthreaded AMIP FINE" # also reported by longruns with a flame graph
+      - label: "Unthreaded AMIP FINE" # also reported by longruns with a flame graph; 10d take 21h, so reducing to 1d
```

Comment:
Not sure if it is worth specifying "unthreaded" now that all runs are unthreaded

Reply:
Good point. n=1 is more accurate here.